docs/common/standards.rst

```diff
@@ -53,7 +53,7 @@ Technical References
       - Lodderstedt, T., Campbell, B., "JWT Secured Authorization Response Mode for OAuth 2.0 (JARM)", November 2022.
     * - :rfc:`6749`
       - The OAuth 2.0 Authorization Framework
-    * - `DPOP`
-      - D. Fett, B. Campbell, J. Bradley, T. Lodderstedt, M. Jones, D. Waite, "OAuth 2.0 Demonstrating Proof-of-Possession at the Application Layer (DPoP)"
-    * - `OPENID4VC-HAIP`
+    * - `DPOP`_
+      - D\. Fett, B. Campbell, J. Bradley, T. Lodderstedt, M. Jones, D. Waite, "OAuth 2.0 Demonstrating Proof-of-Possession at the Application Layer (DPoP)"
```

Comment:
DPoP becomes an RFC as well, shall we update it and add the tag for RFC9449?

Reply:
Yes, please.

Reply:
Done e0248809479b4e122f4653e1f934b8f5c57ce208